docs/guides/plugins.rst

```diff
@@ -49,6 +49,28 @@ constants like ``__DIR__``, its return value should not change. The currently su
  * ``user_settings``
  * ``widgets``
 
+elgg-services.php
```

Comment:
this couldn't be done in `elgg-plugin.php`?

Reply:
It could be in theory, but I like the separation that Steve introduced initially.